.github/workflows/windows.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         platform:
           - arch: win64
             config: enable-fips
+          - arch: win64
```

Comment:
I'm not fluent with workflow .yml file so I might be wrong. However it seems to me the newly added `arch win64` block is in conflict with existing `win64` block at line 24. Is that intentional?

Reply:
No, it does not conflict.